### PR TITLE
Make PREMIS version 3 constants

### PR DESCRIPTION
**Pull request by ablwr**
Making this PR to build and test the SS with. This connects to https://github.com/archivematica/Issues/issues/370.

### CODE REVIEW

**Comment by ablwr** [comment]
This connects to https://github.com/archivematica/Issues/issues/370.

Upgrading from [WIP] because I think its suitable to be reviewed and merged in. It will eventually need to be added and a new version cut for a proper update in the associated PRs for AM and SS, although can be easily worked-around for now by pointing to the commit in this branch.

**Comment by ablwr** [comment]
Agreed!

File: metsrw/utils.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
     "xsi": "http://www.w3.org/2001/XMLSchema-instance",
     "mets": "http://www.loc.gov/METS/",
     "premis": "info:lc/xmlns/premis-v2",
+    "premis3": "http://www.loc.gov/premis/v3",
```
Comment (sevein) [inline metsrw/utils.py]: We don't need this AFAIK.

**Comment by sevein** [comment]
We agreed to close this for now as we don't need it at the moment.

**Pull request closed without merging**